clarifai/models/model_serving/docs/custom_config.md

```diff
@@ -0,0 +1,33 @@
+## Custom Triton Configurations
+
+The commandline triton model repository generation utils do work with default values for the various triton configurations but a few of these config values can be modified to suit different task specific needs.
+
+* For vision models for instance, different input shapes for the `Height (H)` and `Width (W)` are supported and can be set via the commandline too.i.e.
+```console
+$ clarifai-model-upload-init --model_name <Your model name> \
+		--model_type <select model type from available ones> \
+		--image_shape "H, W"
```

Comment:
--image_shape should be `[H, W]` instead `H,W`

Reply:
Both `[H, W]` and `H, W` are accepted by the `dims_type` (defined under repository.py)